hack/e2e-common.sh

```diff
@@ -136,6 +155,22 @@ function install_mpi {
     kubectl apply --server-side -f "${KUBEFLOW_MPI_MANIFEST}"
 }
 
+#$1 - cluster name
+function install_kuberay {
+    # Extra e2e images required for Kuberay, 
```

Comment:
that one gives us the speed on test execution, but takes time here...
I am about to measure how much